### PR TITLE
add relay, radar and some aura components in the list

### PR DESCRIPTION
**Pull request by harikt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,7 +184,8 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
 
 * [Stack](https://github.com/stackphp) - A library of stackable middleware for Silex/Symfony.
 * [Slim Middleware](https://github.com/codeguy/Slim-Middleware) - A collection of custom middleware for Slim.
-* [Conduit](https://github.com/phly/conduit) - A port of [Sencha Connect](https://github.com/senchalabs/connect) to PHP.
+* [Stratigility](https://github.com/zendframework/zend-stratigility) - A port of [Sencha Connect](https://github.com/senchalabs/connect) to PHP.
```
Comment (harikt) [inline README.md]: Conduit is renamed to stratigility.

**Comment by josegonzalez** [comment]
Please resubmit this as separate PRs.

**Comment by harikt** [comment]
@josegonzalez Would love to hear what is the good thing with separate PR's ?

**Comment by josegonzalez** [comment]
I can better classify them, reject individual ones, and have more notifications in my email.

**Comment by harikt** [comment]
Doesn't comments on commit work? So people don't need to spend more time on individual PR?

**Comment by harikt** [comment]
Anyway don't worry, I don't have enough time or energy to send PR for the current time.

**Comment by josegonzalez** [comment]
It's just easier for me to review each pr for each individual contribution and open/close/comment as necessary if it's one thing to review at a time. In this case, I need to:

- look at the provided thing
- see if it overlaps with/duplicates another link
- see what it should replace
- tell the user to fix/unfix something
- close a pr

Easier for me as a maintainer to do it on a single-package basis, as I maybe only have time to look at 5 at a time. I'd rather prioritize looking at a specific section to get those sections down to zero than spend my time on all the contributions by one person in a single pr. If that seems heavy-handed, sorry.

Anyhow, a few of these were already merged/fixed in other PRs, so feel free to submit the other's as one-off as you feel necessary.

**Pull request closed without merging**